examples/nginx-grafana/README.md

```diff
@@ -0,0 +1,340 @@
+nginx-grafana-access_log-1  | Combined all IP addresses into a single file: /tmp/nginx_export.ipinfo.
+nginx-grafana-access_log-1  | removed '/tmp/nginx_export.ipinfo.tmp'
+nginx-grafana-access_log-1  | Extracting the rest of the data from /tmp/nginx_export.csv to /tmp/nginx_import.csv ...
+nginx-grafana-access_log-1  | renamed '/tmp/nginx_export.csv' -> '/tmp/nginx_import.csv'
+nginx-grafana-access_log-1  | Geo-locating all the IP addresses in bulk using the IPInfo API (https://ipinfo.io):
+nginx-grafana-access_log-1  |    Total IP addresses that will be looked up in bulk using /tmp/nginx_export.ipinfo: 10000.
+nginx-grafana-access_log-1  |    Geo-lookup of 10000 IP addresses written to file: /tmp/nginx_import.ipinfo.csv.
+nginx-grafana-access_log-1  |    Complete!
+nginx-grafana-access_log-1 exited with code 0
+nginx-grafana-grafana-1     | ✔ Downloaded and extracted timeplus-proton-datasource v1.0.3 zip successfully to /var/lib/grafana/plugins/timeplus-proton-datasource
+nginx-grafana-grafana-1     | 
+nginx-grafana-grafana-1     | Please restart Grafana after installing or removing plugins. Refer to Grafana documentation for instructions if necessary.
+nginx-grafana-grafana-1     | 
+nginx-grafana-grafana-1     | logger=settings t=2024-07-19T21:43:50.364886485Z level=info msg="Starting Grafana" version=11.1.0 commit=5b85c4c2fcf5d32d4f68aaef345c53096359b2f1 branch=HEAD compiled=2024-07-19T21:43:50Z
+...
+``` 
+
+5. Open http://localhost:3000 in your web browser:
+![](images/05.jpeg)
+
+
+6. Click on the hamburger menu at the top left and click on "Connections" to navigate to the "Connections" page so you confirm that the Timeplus Proton plugin was installed successfully:
+![](images/06.jpeg)
+
+
+7. While on the "Connections" page, type "proton" in the search box then click on the "Proton" data source:
+![](images/07.jpeg)
+
+The Timeplus Proton plugin was correctly installed in the Docker container:
+![](images/07b.jpeg)
+Next click on the "Add new datasource" button at the top right hand corner of the page.
+
+
+8. While on the "Add data source" page, change the host from `localhost` to `proton` then click on the "Save and test" button to confirm that you are able to establish a connection to the Timeplus Proton database server running in Docker:
+![](images/08.jpeg)
+Next, click on the "Build a dashboard" button at the top right hand corner of the page.
+
+9. While on the "New dashboard" page, click on the "Import dashboard" button:
+![](images/09.jpeg)
+
+
+10. Next drag the [`nginx-access-logs_grafana-dashboard.json`](nginx-access-logs_grafana-dashboard.json) file inside the `proton/examples/nginx-grafana` subfolder and drop it on this page to upload it:
+![](images/10.jpeg)
+![](images/10b.jpeg)
+
+11. Now choose the Timeplus Proton data source you setup earlier:
+![](images/11.jpeg)
+
+12. Congrats! You've successfully set up the "Nginx Access Logs Analysis" dashboard in Grafana:
```

Comment:
The steps are quite long. Thanks for the screenshots for each step. This certainly help but maybe you can just pre-configure the docker compose to setup the dashboard and source.  You can check my example at https://github.com/timeplus-io/proton/tree/develop/examples/grafana You may mount it at ./grafana_provisioning:/etc/grafana/provisioning In the folder it can define the Timeplus proton data source and use that source in the dashboard JSON.